### ISSUE TITLE
Turning HW on via Domo switch confuses IOS App

### ISSUE THREAD

**Issue by Malarcy** [imcfarla2003/domoticz-hive#3]
Just installed plugin - all looks ok - but when I set the hotwater "on" via the domoticz switch the ios app errors on data refresh with "it's not you its us - please try again".

From the hive web dashboard the hotwater was set up "manual" and was shown as "On" - resetting it to schedule (from the web dashboard) cleared the error in the ios app.

domo version 3.9174
plugin version  0.5(Dev) (pulled via git today)
platform ubuntu 64bit 16.04 LTS 
hive v2 

Happy to try stuff to debug (when the family aren't waiting on the heating or hot water)

**Comment by imcfarla2003** [comment]
Sorry I don't use ios (political reasons).  Does the web frontend work ok?

**Comment by Malarcy** [comment]
Work ok ?  

Doesn’t crash and I am able to reverse the situation that the plugin created ie I can change settings and the the iOS app works again. 

Ok you don’t use iOS. 

I don’t use android because after 5 years of developing commercial software for the the platform I got tired of the fragmentation but each to their own. 

It’s a non issue anyway for me since i have subsequently looked at what I actually want Domoticz to do and in reality all I want is the ability to set a temp in response to certain other states and I can accomplish that with a python script so I have stepped away from the plugin

It’s good work and I can see others may want the fuller feature set so good luck with it going  forward. 

Thanks for the follow up.
